src/src/AdminBundle/Controller/EventTypeController.php

```diff
@@ -65,6 +69,15 @@ public function editAction($id, Request $request)
             );
         }
 
+        $eventTypeLanguages = $em->getRepository(EventTypeLanguages::class)->findBy(['eventType' => $id]);
+
+        $eventTypeLanguagesIds = [];
+        foreach ($eventTypeLanguages as $eventTypeLanguage) {
+            $eventTypeLanguagesIds[] = $eventTypeLanguage->getLanguage();
+        }
+
+        $languages = $em->getRepository(Language::class)->findById($eventTypeLanguagesIds);
+
```

Comment:
Rado len jedna vec, ak uz mas EventType a mas v entitach namapovane cudzie kluce, tak uz nemusis postupne a jednotlivo cez idecka sa preoracovat k languages, ktore sa viazu k EventTypu. Da sa to urobit takto nejako: 
eventType->getLanguages()